src/clients/Governance.ts

```diff
@@ -762,8 +762,16 @@ export class Governance extends API {
     return response.data
   }
 
-  async getLatestEvents() {
-    const response = await this.fetch<ApiResponse<ActivityTickerEvent[]>>(`/events`, this.options().method('GET'))
+  async getLatestEvents(eventTypes: EventType[]) {
+    let query = eventTypes.map((type) => `event_type=${type}`).join('&')
+    if (query) {
+      query = '?' + query
+    }
```

Comment:
We have multiple ways of achieving this. Check for example the one `getCachedVotesByProposals`. Maybe we could extract this logic to a function.